Mathlib/AlgebraicGeometry/Restrict.lean

```diff
@@ -469,4 +480,29 @@ instance {X Y : Scheme.{u}} (f : X ⟶ Y) (U : Opens Y) [IsOpenImmersion f] :
 
 end MorphismRestrict
 
+/-- If `f` is an open immersion `X ⟶ Y`, the global sections of `X`
+are naturally isomorphic to the sections of `Y` over the image of `f`. -/
+noncomputable
+def IsOpenImmersion.ΓIso {X Y : Scheme.{u}} (f : X ⟶ Y) [IsOpenImmersion f] :
+    Scheme.Γ.obj (op X) ≅ Γ(Y, Scheme.Hom.opensRange f) :=
+  Scheme.Γ.mapIso (IsOpenImmersion.isoOfRangeEq
+      (Scheme.ιOpens (Scheme.Hom.opensRange f)) _ Subtype.range_val).op ≪≫
+    Y.presheaf.mapIso (eqToIso (TopologicalSpace.Opens.openEmbedding_obj_top _).symm).op
```

Comment:
Maybe use `invApp` to define this (we probably want to wrap that into an iso first). And use `Γ(X, ⊤)` instead of `Scheme.Γ.obj`.

Reply:
I did. I am wondering what the simp NF should be for `Scheme.Hom.opensRange f`?

Reply:
`Scheme.Hom.opensRange f` is good enough. we probably should have a simp `f '' U = opensRange`. btw you could write `f.opensRange`.